README.md

```diff
@@ -1,24 +1,25 @@
 # Auth0 Documentation
 This is the repository for the Auth0 documentation.
 
-## Contributing Guidelines
-* Read and follow the [STYLEGUIDE](STYLEGUIDE.md)
-* Consult the [WORDS](WORDS.md) document for Auth0 specific spellings and definitions
+## Contribution Guidelines
+* Read and follow the [Style Guide](STYLEGUIDE.md).
+* Consult the [Words](WORDS.md) document for Auth0 specific spellings and definitions.
 * Always use absolute links from the root of the site. Note, that your links should NOT start with `/docs` or any other base path. If the site is hosted with a base like on `auth0.com/docs` the links will be corrected at runtime.
```

Comment:
Thats what I thought but since that's a relative link, its the "Always use absolute links " that is confusing to me. Just making sure I had it right before changing it.